back/src/whombat/schemas/spectrograms.py

```diff
@@ -114,3 +114,12 @@ class SpectrogramParameters(STFTParameters, AmplitudeParameters):
 
     cmap: str = "gray"
     """Colormap to use for spectrogram."""
+
+    gamma: float = 1.0
+    """Gamma of the resulting spectrogram."""
+
+    low_signal: float = 0.0
+    """Signal strength to set as the minimum."""
+
+    high_signal: float = 1.0
+    """Signal strength to set as the maximun."""
```

Comment:
Could you please add a bit more documentation to this section. Just to make 
sure the reader understands that this new parameters correspond to the
gamma correction of the generated image.